dev/fattest.simplicity/src/componenttest/rules/repeater/JakartaEE11Action.java

```diff
@@ -77,7 +77,7 @@ public class JakartaEE11Action extends FeatureReplacementAction {
                                                   "appAuthorization-3.0",
                                                   "appSecurity-6.0",
                                                   "batch-2.1",
-                                                  "beanValidation-3.0",
+                                                  "beanValidation-3.1",
                                                   "cdi-4.1",
```

Comment:
Should managedBeans-2.0 be removed from this list?

Reply:
So removed.